source/renderer/app/components/staking/stake-pools/StakePoolsSearch.js

```diff
@@ -113,16 +112,15 @@ export class StakePoolsSearch extends Component<Props> {
           {this.hasSearchClearButton && (
             <div className={clearSearchClasses}>
               {this.hasSearchClearButton && (
-                <Tooltip
-                  skin={TooltipSkin}
-                  tip="Clear"
-                  className={styles.clearSearch}
-                  isOpeningUpward={!isClearTooltipOpeningDownward}
+                <PopOver
+                  content="Clear"
+                  contentClassName={styles.clearSearch}
+                  placement={!isClearTooltipOpeningDownward ? 'top': 'bottom'}
```

Comment:
small thing but here it would be clearer to remove the negation: `isClearTooltipOpeningDownward ? 'bottom': 'top`

Reply:
Fixed